npcx/common/reg_type.h

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2020 The Chromium OS Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+#ifndef _NUVOTON_NPCX_REG_TYPE_H
+#define _NUVOTON_NPCX_REG_TYPE_H
+
+#include <stdint.h>
+
+/******************************************************************************/
+/*
+ * Register Type Definitions
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* ADC device instance */
```

Comment:
I think it would be nice to spell out each abbreviation at least one here in the header

Reply:
I will modify it later.